proarc-common/src/main/java/cz/cas/lib/proarc/common/mods/ndk/NdkPageMapper.java

```diff
@@ -148,17 +148,25 @@ public ModsDefinition toMods(Page page, ModsDefinition mods) {
 
         String pageIndex = page.getIndex();
         String pageNumber = page.getNumber();
-        PartDefinition part = new PartDefinition();
-        part.setType(pageType);
-        mods.getPart().add(part);
-        addDetailNumber(pageIndex, NUMBER_TYPE_PAGE_INDEX, part);
-        addDetailNumber(pageNumber, NUMBER_TYPE_PAGE_NUMBER, part);
-
-        if (page.getNote() != null) {
-            Text text = new Text();
-            text.setValue(page.getNote());
-            part.getText().add(text);
+
+        if((pageType != null && !PAGE_TYPE_NORMAL.equals(pageType)) || pageIndex != null || pageNumber != null) {
```

Comment:
Tohle zahodí poznámku pro normální stranu s prázdnými poli index a number. Jestli to prošlo testy, tak prosím doplnit i testy.